daemon/src/wallet.rs

```diff
@@ -117,6 +118,13 @@ impl Actor<ElectrumBlockchain, sled::Tree> {
             "Wallet seed and Electrum RPC client on different networks."
         );
 
+        let wallet_name = wallet_name_from_descriptor(
```

Comment:
🤔 do we lose the ability to run maker and taker in the same directory, or is the result of this function a unique string? previously used `maker-wallet` was covering this by being hard-coded

Reply:
As long as we don't use the same key (which we shouldn't anyway) they can run in the same directory.